src/DynamoCore/Library/LibraryServices.cs

```diff
@@ -476,13 +471,10 @@ internal void LoadLibraryMigrations(string library)
         {
             string fullLibraryName = library;
 
-            if (!pathManager.ResolveLibraryPath(ref fullLibraryName))
-                return;
-
             string migrationsXMLFile = Path.Combine(Path.GetDirectoryName(fullLibraryName),
                 Path.GetFileNameWithoutExtension(fullLibraryName) + ".Migrations.xml");
 
-            if (!File.Exists(migrationsXMLFile))
+            if (!pathManager.ResolveDocumentPath(ref migrationsXMLFile))
```

Comment:
Hi @aosyatnik this fails for locating the RevitNodes.Migrations.xml file since it is in DynamoRevit2015 / 2016 folder and then no migrations are loaded for Revit nodes...